### PR TITLE
FFWEB-1254 Update CSS mixins in 3.0 migration guide

### PR DESCRIPTION
**Pull request by wdutkiewicz-obj**
Documentation update according to CSS changes in WebComponents code.

### CODE REVIEW

File: markdown/3.x/en/documentation/upgrade-guide.md [hunk 1]
```diff
@@ -447,7 +468,33 @@ tagCloudLink
 ffw-tagCloudLink
 ```
 
+
+### Replacement of identifiers
```
Comment (torsten-knauf) [inline markdown/3.x/en/documentation/upgrade-guide.md]:
```suggestion
### Replacement of HTML `id` properties
```

I think _HTML id properties_ is more clear than _identifiers_, isn't it? (same would apply to next comment in the change)

Comment (torsten-knauf) [inline markdown/3.x/en/documentation/upgrade-guide.md]: I am unsure if we need this section. Weren't those elements meant to be hidden by shadow DOM in the past and hence private? If we don't need it, I think we shouldn't bloat this already huge section.

Comment (wdutkiewicz-obj) [reply markdown/3.x/en/documentation/upgrade-guide.md]: You are right, we don't need it. I hadn't realized the ids were encapsulated in the Shadow DOM until I saw your comment. I checked for version 3.0.0 and yes - they're all hidden from the user.

Comment (wdutkiewicz-obj) [reply markdown/3.x/en/documentation/upgrade-guide.md]: This is not relevant any more, as I removed entire section due to your comment.

**Comment by wdutkiewicz-obj** [comment]
@torsten-knauf : please, re-review. I renamed upgrade guide as well to reflect its current state.

**Pull request merged**